editor/export/project_export.cpp

```diff
@@ -1291,27 +1291,6 @@ void ProjectExportDialog::_open_export_template_manager() {
 	EditorNode::get_singleton()->open_export_template_manager();
 }
 
-void ProjectExportDialog::_validate_export_path(const String &p_path) {
```

Comment:
I was not able to determine what this method is for. The same behavior seems to be built in the FileDialog 🤷‍♂️